src/components/Map/Map.css

```diff
@@ -0,0 +1,4 @@
+#map{
```

Comment:
Ids should not be used as a selector in CSS, for the plain reason that IDs should not be reused in the DOM